__tests__/installer.test.ts

```diff
@@ -1,54 +1,58 @@
-import io = require('@actions/io');
-import fs = require('fs');
-import os = require('os');
-import path = require('path');
-
-const toolDir = path.join(
-  __dirname,
-  'runner',
-  path.join(
-    Math.random()
-      .toString(36)
-      .substring(7)
-  ),
-  'tools'
-);
-const tempDir = path.join(
-  __dirname,
-  'runner',
-  path.join(
-    Math.random()
-      .toString(36)
-      .substring(7)
-  ),
-  'temp'
-);
-
-process.env['RUNNER_TOOL_CACHE'] = toolDir;
-process.env['RUNNER_TEMP'] = tempDir;
+import * as io from '@actions/io';
+import * as tc from '@actions/tool-cache';
+import * as fs from 'fs';
+import * as os from 'os';
+import * as path from 'path';
 import * as installer from '../src/installer';
 
-const IS_WINDOWS = process.platform === 'win32';
+const isWindows = process.platform === 'win32';
```

Comment:
this variable had inconsistent naming from the style used everywhere else in this action :)